SimPEG/InvProblem.py

```diff
@@ -113,6 +146,18 @@ def getFields(self, m, store=False, deleteWarmstart=True):
 
         return f
 
+    def get_dpred(self, m, f):
+        if isinstance(self.dmisfit, DataMisfit.BaseDataMisfit):
+            return self.dmisfit.survey.dpred(m, f=f)
+        elif isinstance(self.dmisfit, ObjectiveFunction.BaseObjectiveFunction):
+            dpred = []
+            for i, objfct in enumerate(self.dmisfit.objfcts):
+                if hasattr(objfct, 'survey'):
+                    dpred += [objfct.survey.dpred(m, f=f[i])]
```

Comment:
if you give multiple data misfits, `invProb.dpred` will return you a list the same length as the number of data misfits. 

- [ ] this needs to be tested